macros.in

```diff
@@ -132,6 +132,8 @@
 
 %_keyringpath		%{_dbpath}/pubkeys/
 
+%specpartsdir %{_builddir}/%{buildsubdir}-SPECPARTS
```

Comment:
Move this next to the other build directory stuff, _specdir and buildroot and whatnot.

Reply:
Is buildsubdir always defined?

Reply:
In practise yes, because the debuginfo machinery depends on it.

Reply:
Oh and just realized that moving the specparts dir out of buildsubdir makes it kinda unnecessary to make it user overridable, but maybe there are other use-cases for that anyhow.